agent/pkg/chassis/config/config.go

```diff
@@ -51,11 +59,18 @@ type ConsistentHash struct {
 }
 
 func NewGoChassisConfig() *GoChassisConfig {
+	nodeName, isExist := os.LookupEnv(meshConstants.MY_NODE_NAME)
+	if !isExist {
+		klog.Fatalf("env %s not exist", meshConstants.MY_NODE_NAME)
+		os.Exit(1)
```

Comment:
klog.Fatalf will call the os.Exit. Please delete the os.Exit

Reply:
ok, i will modify